src/encoder.rs

```diff
@@ -1286,6 +1286,26 @@ impl<T: Pixel> FrameInvariants<T> {
     self.input_frameno * TIMESTAMP_BASE_UNIT * self.sequence.time_base.num
       / self.sequence.time_base.den
   }
+
+  /// HDR10+ Metadata as T.35 metadata from [`EncoderConfig`]
+  pub fn hdr10plus_metadata(&self) -> Option<T35> {
+    if !(self.show_frame || self.is_show_existing_frame()) {
+      return None;
+    }
+
+    self
+      .config
+      .hdr10plus_payloads
+      .as_ref()
+      .and_then(|payloads| payloads.get(&self.input_frameno))
+      .and_then(|payload| {
+        Some(T35 {
+          country_code: 0xB5,
+          country_code_extension_byte: 0x00,
+          data: payload.clone().into_boxed_slice(),
```

Comment:
This clone isn't ideal but I'm not sure if it would be better to store `T35` directly in the config, to avoid it.